OpenEXR/IlmImfTest/testFutureProofing.cpp

```diff
@@ -1234,6 +1236,12 @@ modifyType (bool modify_version)
             
             //length of attribute
             fread(&length,4,1,f);
+            if (!GLOBAL_SYSTEM_LITTLE_ENDIAN)
+            {
+                int tmp = bswap_32(length);
```

Comment:
Why a tmp here? Why not just

    length = bswap_32(length);

???

Reply:
It's from 6 years ago and I took it from some other project probably :-) It will work also without the tmp variable I suppose.